frontend/views/components/AvatarUpload.vue

```diff
@@ -105,7 +105,7 @@ export default {
     }
   }
 
-  &-img.is-xl {
+  &-img.is-xl.is-xl { // need .is-xl 2x for specificity
```

Comment:
This is very confusing

Reply:
Yes, indeed. without the 2nd class, these rules are not strong enough. Maybe you could take a look and another way (besides `!important`)

Reply:
I never used that hack! But this is really !important ;)